selfswab/tasks.py

```diff
@@ -35,6 +37,8 @@ def poll_meditech_api_for_results():
                 url=settings.MEDITECH_URL,
                 headers={"Content-Type": "application/json"},
                 json={"barcodes": barcodes},
+                auth=(settings.MEDITECH_USER, settings.MEDITECH_PASSWORD),
+                verify=False,
```

Comment:
We should definitely be verifying the SSL certificate. Why are we not doing it in this case?

Reply:
I just copied it from ndoh-hub where we do jembi calls. I'll remove that and merge, then take it from there

Reply:
Yeah, Jembi didn't have a valid HTTPS certificate for their domain, so we had to disable it there. Not ideal but we didn't have another choice.